src/interfaces/interfaces.ts

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+export interface systemInformation {
```

Comment:
All interface names start with small letters which doesn't match the convention of using PascalCase for interfaces